tests/integration_tests/nlu/test_train.py

```diff
@@ -0,0 +1,60 @@
+import pytest
```

Comment:
sorry, this changed a bit when doing the test suite split. Can we keep it in the unit tests but make sure it's part of `category_full_model_training` in [here](https://github.com/RasaHQ/rasa/blob/d862768082a7d582da13f8f88bdf552c414defa5/tests/conftest.py#L81)?